CalibTracker/SiStripLorentzAngle/src/LA_Filler.cc

```diff
@@ -74,22 +75,25 @@ granularity(const SiStripDetId detid, const float tthetaL, const Long64_t TFE_in
   gran += subdetLabel(detid);
   if(byLayer_)  gran *= layerLabel(detid);
   if(byModule_) gran *= moduleLabel(detid);
-  if(localYbin_) gran += (localy < 0 ? "_yM":"_yP") + boost::lexical_cast<std::string>(abs((int)(localy/localYbin_+(localy<0?-1:0))));
-  if(stripsPerBin_) gran += "_strip"+boost::lexical_cast<std::string>((unsigned)((0.5+((apvstrip/64)?(127-apvstrip):apvstrip)/stripsPerBin_)*stripsPerBin_) );
+  if(localYbin_) gran += (localy < 0 ? "_yM":"_yP") + std::to_string(abs((int)(localy/localYbin_+(localy<0?-1:0))));
+  if(stripsPerBin_) gran += "_strip"+std::to_string((unsigned)((0.5+((apvstrip/64)?(127-apvstrip):apvstrip)/stripsPerBin_)*stripsPerBin_) );
   if(ensembleBins_) {
-    gran+= "_ensembleBin"+boost::lexical_cast<std::string>((int)(ensembleBins_*(tthetaL-ensembleLow_)/(ensembleUp_-ensembleLow_)));
```

Comment:
@alesaggio, as for the merge conflicts, it's my bad:  #19645 has been merged first (though not fully signed). 
To avoid the conflict you can either revert commit 29cea77425adf94ed627ab3bbf6e7a3a3ca7734e, or rebase the branch following the instructions at http://cms-sw.github.io/tutorial-resolve-conflicts.html